api/control.proto

```diff
@@ -436,3 +480,75 @@ message RemoveSecretRequest {
 // RemoveSecretResponse is an empty object indicating the successful removal of
 // a secret.
 message RemoveSecretResponse {}
+
+// GetConfigRequest is the request to get a `Secret` object given a config id.
+message GetConfigRequest {
+	string config_id = 1;
+}
+
+// GetConfigResponse contains the Secret corresponding to the id in
```

Comment:
"... contains the Config..."
The rest of the comment about `Secret.Spec.Data` can probably deleted, since the config data doesn't need to be redacted.